CHANGES.md

```diff
@@ -1,3 +1,9 @@
+# 0.20.0
+
+-   Added a warning hook in the fieldAccessors, allowing you to define
+    warning that can exist next to the erros. The form state now includes
```

Comment:
"a warning"   "erros" -> "errors"

I like "that can exist in addition to the errors" better.